changelogs/fragments/fix_asa_ogs_bug_165_166.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
if looks to be a change affecting tbehaviour of provided input , I believe this should be a major change then , wdyt?

Reply:
The existing play will continue to function as expected because the playbook `str/input` type has not changed. 
Major change is NR in this case, as I had already confirmed it earlier.